README.md

```diff
@@ -17,8 +17,7 @@ All library issues should go to the [issue tracker from github](https://github.c
 
 Compatibility
 -------------
-This release is compatible with all Elasticsearch 7.0 releases and onwards.
-It was tested with version 7.3.0.
+This release is compatible with all Elasticsearch 7.0 releases and onwards. The testsuite is run against the most recent minor version of the Elastic Stack, currently 7.10.2.
```

Comment:
IIRC we only test it against the latest version of "Elasticsearch", not Elastic Stack :)
Please keep the two sentences on two different lines, will help to keep the diff short